views/_components/grouped-typeahead.blade.php

```diff
@@ -6,7 +6,10 @@
     $classes['input'] = $classes['input'] ?? FormComponentsServiceProvider::INPUT_DEFAULT_CLASS;
     $classes['select'] = $classes['select'] ?? FormComponentsServiceProvider::SELECT_DEFAULT_CLASS;
 
-    $value = $value ?? null;
+    $countryValue = $value ?? old($countryName);
```

Comment:
Should this not be a bit more generic as this element could possibly not be used just for a country select?

Reply:
I did think that, it references `$countryName` in some other code so was trying not to refactor it all